### PR TITLE
News character

### PR DESCRIPTION
**Pull request by AnassAmh**

### CODE REVIEW

**Review by Kileana27 — APPROVED** [review]

File: src/app/app.routes.ts [hunk 1]
```diff
@@ -71,6 +74,12 @@ export const routes: Routes = [
     path: 'erreur', 
     component: ErrorPageComponent 
   },
+
+  { 
+    path: 'Character', component: NewsCharacterComponent
```
Comment (Benja-san) [inline src/app/app.routes.ts:79]: french content, english code

File: src/app/pages/news-character/news-character.component.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<div class="character-selection-container">
```
Comment (Benja-san) [inline src/app/pages/news-character/news-character.component.html:1]: section tag

File: src/app/pages/news-character/news-character.component.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  <div class="selection-box">
+    <div class="add-character">
+      <button class="add-button">+</button>
+      <span>Ajouter un personnage</span>
```
Comment (Benja-san) [inline src/app/pages/news-character/news-character.component.html:5]: h1 tag

File: src/app/pages/news-character/news-character.component.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <ul class="slot-column">
+          <li class="slot"></li>
+          <li class="slot"></li>
+          <li class="slot"></li>
+          <li class="slot"></li>
+          <li class="slot"></li>
+        </ul>
+      </div>
+    </div>
+    <div class="scroll-arrow" (click)="scroll('down')">▼</div>
```
Comment (Benja-san) [inline src/app/pages/news-character/news-character.component.html:23]: button if click action

File: src/app/pages/news-character/news-character.component.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  <div class="selection-box">
+    <div class="add-character">
+      <button class="add-button">+</button>
+      <span>Ajouter un personnage</span>
+    </div>
+    <p>Emplacement libre, vous pouvez créer un nouveau personnage</p>
+    <button class="new-character-button">Nouveau personnage</button>
+  </div>
+  <div class="character-slots-container">
+    <div class="scroll-arrow" (click)="scroll('up')">▲</div>
```
Comment (Benja-san) [inline src/app/pages/news-character/news-character.component.html:11]: button if click action

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**